src/js/components/FormField/doc.js

```diff
@@ -17,6 +17,11 @@ export const doc = FormField => {
     .intrinsicElement('div');
 
   DocumentedFormField.propTypes = {
+    a11yTitle: PropTypes.string.description(
+      `Custom label to be used by screen readers. When provided,
+        an aria-label will be added to the element. Should be provided
+        when FormField does not have a child with a11yTitle property.`,
+    ),
```

Comment:
This sentence needs refinement for FormField, It's not accurate to say `When provided,
        an aria-label will be added to the element`, It is only true to when there is no child. try to refine the sentence.